api/src/mailbox.rs

```diff
@@ -590,6 +604,28 @@ impl Request for InvokeDpeReq {
     type Resp = InvokeDpeResp;
 }
 
+// EXTEND_PCR
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct ExtendPcrReq {
+    pub hdr: MailboxReqHeader,
+    pub pcr_idx: u32,
+    pub data: [u8; 48],
+}
+pub type ExtendPcrReqErr = ();
+impl ExtendPcrReq {}
```

Comment:
Unused?
Also it might be a good idea to implement the Request trait like other mailbox commands?